stats_test.go

```diff
@@ -208,3 +234,187 @@ func TestNilAggregation(t *testing.T) {
 		t.Errorf("Expected: %v, Got: %v", fmt.Errorf("aggregation *view.Aggregation is not supported"), actual)
 	}
 }
+
+func TestPercentileName(t *testing.T) {
+	testCases := []struct {
+		Percentile float64
+		Expected   string
+	}{
+		{
+			0.5,
+			"50percentile",
+		},
+		{
+			0.75,
+			"75percentile",
+		},
+		{
+			0.92,
+			"92percentile",
+		},
+		{
+			0.95,
+			"95percentile",
+		},
+		{
+			0.99,
+			"99percentile",
+		},
+		{
+			0.995,
+			"995percentile",
+		},
+		{
+			0.999,
+			"999percentile",
+		},
+	}
+
```

Comment:
And this one please.

Reply:
I think this one should be kept, make sense to have some vertical spacing between the setup and the code that run the test cases.